code/src/components/Details.js

```diff
@@ -0,0 +1,91 @@
+import React, { useEffect, useState } from "react";
+import { useParams, Route } from "react-router-dom";
+import { useHistory } from "react-router";
+import Loader from "./Loader";
+import { DETAIL_URL } from "../utils/urls";
+
+import "./Details.css";
+import NotFound from "./NotFound";
+
+const Details = () => {
+  const [movieDetail, setMovieDetail] = useState([]);
+  const [notFound, setNotFound] = useState(false); // state property to load not found page if movie id is incorrect
+  const [loading, setLoading] = useState(false); // state property for loader intialized with false
+
```

Comment:
The setLoading , shouldn’t that be set to true so when the page is waiting to get data from the api the loading shows?